modules/ROOT/pages/ext-analytics-elk.adoc

```diff
@@ -0,0 +1,173 @@
+= Analyze Business and API Data using ELK
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Use the Elastic Stack (ELK) to analyze the business data and API analytics generated by Anypoint Platform Private Cloud Edition (Anypoint Platform PCE). You can use Filebeat to process Anypoint Platform log files, insert them into an Elasticsearch database, and then analyze them with Kibana.
+
+The following image shows an example of how API data appears in Kibana:
+
+image::kibana-example.png[]
+
+== Prerequisites
+
+* Required software
++
+To export external analytics data to ELK, you must have the following software:
++
+** Mule Runtime, version 3.8.4 or later
+** Mule Runtime Manager Agent, version 1.7.0 or later
+** Elasticsearch, version 7.6.2
+** Filebeat, version 7.6.2
+** Kibana, version 7.6.2
+
+* Ensure that you have installed and configured your Mule runtime engine instances.
+* Ensure that you have registered these instances with Runtime Manager.
+
+== Edit the wrapper.conf Properties File
+
+In your Mule runtime installation, edit the following properties in the `conf/wrapper.conf` file:
+
+. Set the `analytics_enabled` property to `true`.
```

Comment:
GLOBALLY end text that introduces an example, list, table, and so on with a colon:
```suggestion
. Set the `analytics_enabled` property to `true`:
```

Reply:
Please repeat this punctuation for each step that follows.